coral/src/app/router_utils.ts

```diff
@@ -54,14 +54,14 @@ const TOPIC_OVERVIEW_TAB_ID_INTO_PATH = {
   [TopicOverviewTabEnum.ACLS]: "subscriptions",
   [TopicOverviewTabEnum.MESSAGES]: "messages",
   [TopicOverviewTabEnum.SCHEMA]: "schema",
-  [TopicOverviewTabEnum.DOCUMENTATION]: "documentation",
+  [TopicOverviewTabEnum.DOCUMENTATION]: "readme",
```

Comment:
I synced with Roope and we decided to keep the notion of "documentation" for us, as in, in backend as well as in frontend code we're always talking about "documentation" (*) and only change that to `readme` when it's user facing. So I thought it's better to keep the enum named "documentation". 

(*) it's still named "documentation" in the API etc. so we thought that's the best way to avoid confusing between back- and frontend domain

Reply:
sounds good to me